rcl/test/rcl/test_subscription.cpp

```diff
@@ -178,14 +177,15 @@ TEST_F(CLASSNAME(TestSubscriptionFixture, RMW_IMPLEMENTATION), test_subscription
   {
     test_msgs__msg__BasicTypes msg;
     test_msgs__msg__BasicTypes__init(&msg);
-    OSRF_TESTING_TOOLS_CPP_SCOPE_EXIT(
-    {
-      test_msgs__msg__BasicTypes__fini(&msg);
-    });
+    OSRF_TESTING_TOOLS_CPP_SCOPE_EXIT({
```

Comment:
I'm not sure, but I think because of a recent change to the linter (https://github.com/ament/ament_lint/pull/210) that this will cause an error. We should put the brace on the next line:
```suggestion
    OSRF_TESTING_TOOLS_CPP_SCOPE_EXIT(
    {
```